lib/govuk_app_config/configure.rb

```diff
@@ -33,4 +33,8 @@
   config.transport_failure_callback = Proc.new {
     GovukStatsd.increment("error_reports_failed")
   }
+
+  # This stops exceptions rescued by rails from appearing in Sentry.
+  # See https://www.rubydoc.info/gems/sentry-raven/1.2.2/Raven%2FConfiguration:rails_report_rescued_exceptions
+  config.rails_report_rescued_exceptions = false
```

Comment:
I think the introduction of this config option means we can remove quite a few exceptions from the `excluded_exceptions` list. There's a list slightly further down in this section: https://guides.rubyonrails.org/configuring.html#configuring-action-dispatch

```
config.action_dispatch.rescue_responses = {
  'ActionController::RoutingError'               => :not_found,
  'AbstractController::ActionNotFound'           => :not_found,
  'ActionController::MethodNotAllowed'           => :method_not_allowed,
  'ActionController::UnknownHttpMethod'          => :method_not_allowed,
  'ActionController::NotImplemented'             => :not_implemented,
  'ActionController::UnknownFormat'              => :not_acceptable,
  'ActionController::InvalidAuthenticityToken'   => :unprocessable_entity,
  'ActionController::InvalidCrossOriginRequest'  => :unprocessable_entity,
  'ActionDispatch::Http::Parameters::ParseError' => :bad_request,
  'ActionController::BadRequest'                 => :bad_request,
  'ActionController::ParameterMissing'           => :bad_request,
  'Rack::QueryParser::ParameterTypeError'        => :bad_request,
  'Rack::QueryParser::InvalidParameterError'     => :bad_request,
  'ActiveRecord::RecordNotFound'                 => :not_found,
  'ActiveRecord::StaleObjectError'               => :conflict,
  'ActiveRecord::RecordInvalid'                  => :unprocessable_entity,
  'ActiveRecord::RecordNotSaved'                 => :unprocessable_entity
}
```

Reply:
I'd like to do that in a separate PR. Or do you think we should do it here?

Reply:
Hmm, I would probably do it in this PR because the behaviour before and after should be the same. But it doesn't really matter I suppose.

Reply:
That's cool. I'll do it here. 😉